ocs_ci/utility/utils.py

```diff
@@ -352,21 +352,45 @@ def custom_ceph_config(suite_config, custom_config, custom_config_file):
     return full_custom_config
 
 
+def mask_secrets(plaintext, secrets):
+    """
+    Replace secrets in plaintext with asterisks
+
+    Args:
+        plaintext (str): The plaintext to remove the secrets from
+        secrets (list): List of secret strings to replace in the plaintext
+
+    Returns:
+        str: The censored version of plaintext
+
+    """
+    for secret in secrets:
+        plaintext = plaintext.replace(secret, '*' * 5)
+    return plaintext
+
+
 def run_cmd(cmd, **kwargs):
     """
     Run an arbitrary command locally
 
     Args:
         cmd (str): command to run
 
+    Keyword Args:
+        secrets (list): The list of secrets to be masked with asterisks
+            This kwarg is popped in order to not interfere with
```

Comment:
I find this is hidden thing for users but..

Reply:
This seems very prone to error. Can we be 100% sure that we pop the correct `kwarg` off so we don't interfere with what might be being passed to subprocess? What is stopping `secrets` from being it's own arg so we don't hit this issue?

Reply:
Not reason at all to prefer **kwargs. Fixed, thank you :)